### ISSUE TITLE
PFAChain Pool Init for not initialized pool

### ISSUE THREAD

**Issue by patrick-boueri** [opendatagroup/hadrian#7]
https://github.com/opendatagroup/hadrian/blob/ad5b1f731593a2524abc2eabe647f9f07a32ce1c/titus/titus/producer/chain.py#L636

If there is a pool, there's an issue that if it's not initialized it's unable to zip and will throw an error. Willing to provide more details if necessary

**Comment by jpivarski** [comment]
Yes, please provide more details. If you can point to an example pair of PFAs that `pfachain` throws an error rather than producing a chained PFA, it would be much easier to see what needs to be fixed.

**Comment by patrick-boueri** [comment]
Will look into generating an example error to share
